src/components/paragraph/docs.mdx

```diff
@@ -0,0 +1,27 @@
+---
+componentName: Paragraph
+---
+
+The `Paragraph` component should be used for easily styling `p` elements to conform to the design system.
```

Comment:
💭  Along this thought ☝️ , I wonder if turning this into a `Typography`/`Text` component and then supporting an `as` prop like @didoo mentioned would give us the flexibility we're looking for.

```jsx
<Typography as='p'>
<Typography as='span'>
<Typography as='li'>
```

Reply:
I'm definitely for updating this to be a more general component! I think `Text` will make a great name. Since there are some places where generalizing this component will make other existing code simpler, I'm going to create a follow-up PR for renaming the component and making those changes.

Reply:
@ashleemboyer if you don't mind, can you mention me in the follow-up PR for the `<Text>` component? it would be great if we could coordinate the component APIs so I can open an equivalent PR for a similar component for Ember (even though is more controversial in that case)

Reply:
I can do that @didoo!